gateway/crates/federated-server/src/server/graph_updater.rs

```diff
@@ -46,15 +46,22 @@ pub(super) struct GraphUpdater {
     otel_reload: Option<oneshot::Sender<OtelReload>>,
 }
 
-#[derive(Debug, serde::Deserialize)]
+#[derive(Debug, serde::Serialize, serde::Deserialize)]
 #[allow(dead_code)]
-struct UplinkResponse {
-    account_id: Ulid,
-    graph_id: Ulid,
-    branch: String,
-    branch_id: Ulid,
-    sdl: String,
-    version_id: Ulid,
+/// Response from the API containing graph information
+pub struct UplinkResponse {
```

Comment:
We should use GDN, not Uplink in our codebase.

Reply:
https://linear.app/grafbase/issue/GB-6453/rename-uplink-to-gdn-in-the-grafbase-repo-codebase